.prettierrc

```diff
@@ -1,5 +1,13 @@
 {
   "singleQuote": true,
   "trailingComma": "es5",
-  "arrowParens": "avoid"
+  "arrowParens": "avoid",
+  "overrides": [
```

Comment:
Apparently newer versions of Prettier add trailing commas to jsonc files which is not always supported.

Reply:
I think this applies to `.json` files as well, no? I know `package.json` is not happy if you add a trailing comma